### PR TITLE
[15.0][MIG] sale_stock_return_request: Migration to 15.0

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
@Tecnativa TT36544

### CODE REVIEW

**Comment by cesar-tecnativa** [comment]
Deppends on https://github.com/OCA/stock-logistics-workflow/pull/1032

**Comment by sergio-teruel** [comment]
Depends on:
- [x] https://github.com/OCA/stock-logistics-workflow/pull/1032

**Review by chienandalu — APPROVED** [review]
Thanks!

File: sale_stock_return_request/tests/test_stock_return_request.py [hunk 1]
```diff
@@ -151,14 +151,12 @@ def test_01_return_sale_stock_from_customer(self):
...
-                "domain": "[('id', 'in', %s)]" % (sorted(sale_orders.ids)),
-            },
-            res,
-        )
+        expected_res = {
+            "type": "ir.actions.act_window",
+            "res_model": "sale.order",
+            "domain": "[('id', 'in', %s)]" % (sorted(sale_orders.ids)),
+        }
+        self.assertEqual(res, {**res, **expected_res})
```
Comment (pedrobaeza) [inline sale_stock_return_request/tests/test_stock_return_request.py:162]: I don't get this assert.

Comment (chienandalu) [reply sale_stock_return_request/tests/test_stock_return_request.py:162]: Yes, I had to check it as well, but it makes sense in the context of `assertDictContainsSubset()` being now obsolete (https://stackoverflow.com/a/59777678/3274287)

Comment (pedrobaeza) [reply sale_stock_return_request/tests/test_stock_return_request.py:162]: At least a comment is needed to understand this line and the other that is the same.

Comment (chienandalu) [reply sale_stock_return_request/tests/test_stock_return_request.py:162]: Ok. Please do @stefan-tecnativa :)

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**